src/Gui/BitmapFactory.cpp

```diff
@@ -304,21 +311,15 @@ QPixmap BitmapFactoryInst::pixmapFromSvg(const char* name, const QSizeF& size,
         QFile file(iconPath);
         if (file.open(QFile::ReadOnly | QFile::Text)) {
             QByteArray content = file.readAll();
-            icon = pixmapFromSvg(content, size, colorMapping);
+            icon = pixmapFromSvg(content, QSizeF{size.width() * dpr, size.height() * dpr}, colorMapping);
         }
     }
 
-    return icon;
-}
+    if (!icon.isNull()) {
+        icon.setDevicePixelRatio(dpr);

```

Comment:
This doesn't need to be before render?

Reply:
Wdym by render?
If I put it before
```icon = pixmapFromSvg(content, QSizeF{size.width() * dpr, size.height() * dpr}, colorMapping);``` 
then I think it will probably be overwritten right?

Reply:
The docs don't mention anything about this.